sender.go

```diff
@@ -52,6 +52,7 @@ type (
 		partitionID     *string
 		Name            string
 		recoveryBackoff *backoff.Backoff
+		maxRetries      int
```

Comment:
Should this an the above be replaced with a `senderRetryOptions` like in `Hub`?

Reply:
Done.